Sources/xcresultparser/CoberturaCoverageConverter.swift

```diff
@@ -31,7 +31,12 @@
 import Foundation
 import XCResultKit
 
-public class CoberturaCoverageConverter: CoverageConverter {
+public class CoberturaCoverageConverter: CoverageConverter, XmlSerializable {
+
+    public var xmlString: String {
+        return try! xmlString(quiet: true)
```

Comment:
I think for now a simple string is fine. In my case my toolchain reports me a nice error if there is an xml parsing issue. But in general you are right. Once more output formats are added this could be improved further.